modules/openapi-generator/src/main/resources/ruby-client/base_object.mustache

```diff
@@ -93,7 +95,7 @@
           is_nullable = self.class.openapi_nullable.include?(attr)
           next if !is_nullable || (is_nullable && !instance_variable_defined?(:"@#{attr}"))
         end
-        
+
```

Comment:
Hmm I think you need to do the same check you did [here](https://github.com/OpenAPITools/openapi-generator/pull/5706/files?file-filters%5B%5D=.mustache#diff-488b370aef6f37384cf8676c79a431685e7f9ca1c6cd2796e4a746e9a0f7e91fR142-R144)  in `api_client.mustache` for `build` vs `build_from_hash` in the `_deserialize` method of `base_object` in case the oneOf is a property of a regular object.